lambdas/es/indexer/index.py

```diff
@@ -48,6 +48,9 @@
 #  a custom user agent enables said filtration
 USER_AGENT_EXTRA = " quilt3-lambdas-es-indexer"
 
+# Key prefixes that can be ignored if 403 responses are returned when accessing
+FORBIDDEN_PREFIXES = tuple(filter(None, os.environ.get('FORBIDDEN_PREFIXES', "").split(" ")))
```

Comment:
Prefer list comprehension over `filter`. The `or` is a bit safer than `.get(X, DEFAULT)` because if we set `FORBIDDEN_PREFIXES=""` it will forbid everything... probably dangerous.

```suggestion
FORBIDDEN_PREFIXES = [int(x) for x in (os.environ.get("FORBIDDEN_PREFIXES") or "402 403 404").split(" ")]  
```